tensorboard/data/server/Cargo.toml

```diff
@@ -113,3 +114,12 @@ additional_flags = [
 
 [package.metadata.raze.crates.prost-build.'0.7.0']
 gen_buildrs = true
+
+[package.metadata.raze.crates.tonic-reflection.'0.1.0']
+gen_buildrs = true
+build_data_dependencies = ["@com_google_protobuf//:protoc"]
+
+[package.metadata.raze.crates.tonic-reflection.'0.1.0'.buildrs_additional_environment_variables]
+PROTOC = "$(execpath @com_google_protobuf//:protoc)"
+# we don't have easy access to a rustfmt binary, so make it a no-op
```

Comment:
Maybe worth dropping a link to https://github.com/bazelbuild/rules_rust/issues/87 in case this someday is easier?

Reply:
Done. For additional reference, bazelbuild/rules_rust#479 is also worth
knowing about, but not directly relevant to this comment.